content/_help/verify-your-identity/phone-number._zh.md

```diff
@@ -19,7 +19,7 @@ order: 5
 
 ## 你不能使用以下信息来验证身份：
 - 国际电话号码以及有些美国属地的号码
-- 谷歌语音或类似 VOIP (基于IP的语音传输) 号码
+- 谷歌语音或类似VoIP（基于IP的语音传输）号码
```

Comment:
Just double-checking the removal of the space before "VoIP" is intentional?

Reply:
Yes, that is how it is formatted in the [new translations](https://docs.google.com/spreadsheets/d/1QjOnQmMuq796o3W3EnAB1RqY_InMfD5qtZQE146s97w/edit?gid=0#gid=0)